src/particles/elements/All.H

```diff
@@ -22,6 +22,7 @@
 #include "Sbend.H"
 #include "ShortRF.H"
 #include "Sol.H"
+#include "PROT.H"
```

Comment:
Shall we maybe abbreviate this as
```suggestion
#include "PRot.H"
```
everywhere?

Reply:
I think using "PRot" is fine, and it makes sense.  The element name "prot" (lower- or upper-case) comes from the code MaryLie, which is why I used it here, and it is sometimes read as "plane rotation" instead of "pole-face rotation".  Another option is to just call it "Rotation".